models/yang/annotations/sonic-fdb-annot.yang

```diff
@@ -0,0 +1,17 @@
+module sonic-fdb-annot {
+
+    yang-version "";
+
+    namespace "http://openconfig.net/yang/annotation";
+    prefix "sfdb-annot";
+
+    import ietf-yang-types { prefix yang; }
```

Comment:
Why do we need this sonic YANG annotation?

Reply:
The file is removed.